packages/instantsearch.js/src/connectors/infinite-hits/connectInfiniteHits.ts

```diff
@@ -85,6 +85,12 @@ export type InfiniteHitsConnectorParams<
   cache?: InfiniteHitsCache<THit>;
 };
 
+type Banner = NonNullable<
+  NonNullable<
+    Required<SearchResults<Hit>['renderingContent']>
+  >['widgets']['banners']
+>[number];
+
```

Comment:
Should this definition (and the same definition in `connectHits.ts`) be moved to `packages/instantsearch.js/src/types.results.ts`?

Reply:
Yes 👍
Looks good otherwise